fpx/src/api/client.rs

```diff
@@ -113,6 +140,6 @@ impl ApiClient {
     ) -> Result<Vec<models::Span>, ApiClientError<SpanGetError>> {
         let path = format!("api/traces/{}/spans", trace_id.as_ref());
 
-        self.do_req(Method::GET, path).await
+        self.do_req(Method::GET, path, None::<()>).await
```

Comment:
Not a big fan of having to define these `::<()>` things, but this is fine for now. We can solve this another time.

Reply:
Wasn't a fan either, but couldn't figure out a way to make it compile without these type annotations :(